src/test/java/uk/gov/pay/adminusers/resources/ServiceResourceStripeAgreementTest.java

```diff
@@ -76,9 +75,23 @@ public void shouldReturn_400_whenProvidedInvalidIPAddress() {
                 .body(payload)
                 .post(format("/v1/api/services/%s/stripe-agreement", service.getExternalId()))
                 .then()
-                .statusCode(400)
+                .statusCode(422)
```

Comment:
where does the 422 status code come from for this?

Reply:
The Hibernate validation which is invoked with the `@ValidIpAddress` returns 422 if a parameter fails validation because it is an invalid entity. This seems appropriate given its description:
https://httpstatuses.com/422

Reply:
Makes sense. In that case, I think you just need to rename this test method and the test below it.